pkg/manifest/os.go

```diff
@@ -793,7 +794,9 @@ func (p *OS) serialize() osbuild.Pipeline {
 		}))
 	}
 
+	logger, _ := syslog.NewLogger(syslog.LOG_DEBUG, 0)
```

Comment:
This looks like a debugging leftover, which should be removed.

Reply:
Busted! :-) Sure that was not intentional.